docs/getting-started.md

```diff
@@ -159,9 +132,7 @@ argocd: v2.0.1+33eaf11.dirty
 
 ### Exposing ArgoCD
 
-To access the ArgoCD running in your Kubernetes cluster, we simply need leverage port-forwarding.
-
-To do so, first capture the service name in an environment variable.
+To access ArgoCD running in your Kubernetes cluster, we can leverage [Kubernetes Port Forwarding](https://kubernetes.io/docs/tasks/access-application-cluster/port-forward-access-application-cluster/). To do so, first capture the ArgoCD service name in an environment variable.
 
 ```
 export ARGO_SERVER=$(kubectl get svc -n argocd -l app.kubernetes.io/name=argocd-server -o name) 
```

Comment:
We should really fix the role produced by the argo addon. Why is it non-deterministic? All docs refer to explicit service name, like argocd-server. It creates problems for Andrew for this bootstrapping task as well.